pi/imu.py

```diff
@@ -1,59 +1,89 @@
+# SPDX-FileCopyrightText: 2020 Bryan Siepert, written for Adafruit Industries
+#
+# SPDX-License-Identifier: Unlicense
 import time
 import board
 import busio
-import adafruit_bno08x
-from adafruit_bno08x.i2c import BNO08X_I2C
 import json
 import socket
+
 from adafruit_bno08x import (
-    BNO_REPORT_GAME_ROTATION_VECTOR,
-    BNO_REPORT_LINEAR_ACCELERATION
+    BNO_REPORT_ACCELEROMETER,
+    BNO_REPORT_GYROSCOPE,
+    BNO_REPORT_MAGNETOMETER,
+    BNO_REPORT_ROTATION_VECTOR,
 )
+from adafruit_bno08x.i2c import BNO08X_I2C
+
+try:
+    i2c = busio.I2C(board.SCL, board.SDA, frequency=400000)
+    bno = BNO08X_I2C(i2c)
+
+    bno.enable_feature(BNO_REPORT_ACCELEROMETER)
+    bno.enable_feature(BNO_REPORT_GYROSCOPE)
+    bno.enable_feature(BNO_REPORT_MAGNETOMETER)
+    bno.enable_feature(BNO_REPORT_ROTATION_VECTOR)
+except Exception as e:
+    raise RuntimeError("Could not initialize IMU")
 
 HOST = "192.168.0.102"  # The server's hostname or IP address
 PORT = 2049  # The port used by the server
-CONTROL_LOOP_FREQ = 5  # Hz
+CONTROL_LOOP_FREQ = 100  # Hz
```

Comment:
Hey, not sure we need that frequent of data and concerned about the effecto on performance with this number. Can we either, 
a. Set it to something more reasonable
b. Make it configurable without a hard change to this file. Maybe instantiating it from elsewhere with variables if possible

Reply:
i just set it back to 5. chances are, this isn't something we will or should be changing often (ever), so it's not worth make it configurable, nor even particularly good practice.